pootle/apps/pootle_store/unit/results.py

```diff
@@ -22,6 +22,11 @@
 class UnitResult(UnitProxy):
 
     @property
+    def nplurals(self):
+        return self.unit[
+            "store__translation_project__language__nplurals"] or 0
```

Comment:
@julen im not really sure what it should be but im reluctant to make it different to the default on the model - https://github.com/translate/pootle/blob/master/pootle/apps/pootle_language/models.py#L92